awx/ui_next/src/screens/Inventory/InventoryDetail/InventoryDetail.jsx

```diff
@@ -74,7 +74,11 @@ function InventoryDetail({ inventory, i18n }) {
           fullWidth
           label={i18n._(t`Instance Groups`)}
           value={
-            <ChipGroup numChips={5}>
+            <ChipGroup
```

Comment:
Is there a reason why some of these views use the PatternFly ChipGroup and others use our new custom ChipGroup?

Reply:
They all should be using our new custom ChipGroup. Let me verify this.

Reply:
I missed that one. Thanks @marshmalien